banking-bench/src/main.rs

```diff
@@ -435,10 +435,10 @@ fn main() {
             ClusterInfo::new(node.info, keypair, SocketAddrSpace::Unspecified)
         };
         let cluster_info = Arc::new(cluster_info);
-        let tpu_use_quic = matches.is_present("tpu_use_quic");
```

Comment:
this always fails because "tpu_use_quic" is not a registered argument with matches.